dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/RegistryConfig.java

```diff
@@ -132,8 +132,8 @@
      */
     private Boolean simple;
     /**
-     * After simplify the registry, should add some paramter individually.
-     * addionalParameterKeys = addParamKeys
+     * After simplify the registry, should add some parameter individually.
+     * additionalParameterKeys = addParamKeys

```

Comment:
There are some other `addionalParameterKeys` and `paramter` typos, you can search them globally and fix them.

Reply:
I have fixed  and modified the name of the unused method

Reply:
Looks good to me.